db/schema.rb

```diff
@@ -33,8 +33,10 @@
   create_table "loans", force: :cascade do |t|
     t.integer  "project_id"
     t.integer  "order_id"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at",     null: false
+    t.datetime "updated_at",     null: false
+    t.integer  "quantity"
+    t.integer  "line_item_cost"
```

Comment:
So I thought that loans (previously Order Items) only have project_id and order_id? We should discuss this as a group?